src/modules/settings/CssVariableSetting.ts

```diff
@@ -1,14 +1,16 @@
 import Setting from './Setting';
+import Requirement from '../requirements/Requirement';
 
 export default class CssVariableSetting extends Setting<string> {
-    constructor(name: string, displayName: string, options = [], defaultValue = '') {
+    constructor(name: string, displayName: string, options = [], defaultValue = '', requirements: Requirement[] = []) {
```

Comment:
Requirements should just be a requirement instead of a list of requirements.
I know we have some lists of requirements out there, but that is a mix of them being added before MultiRequirement, and people (like me) not knowing what the standard should be